File: Pixly-BE/Pixly.Models/DTOs/Report.cs
namespace Pixly.Models.DTOs
{
    public class Report
    {
        public string ReportMessage { get; set; }
        public string ReportTitle { get; set; }
        public User ReportedByUser { get; set; }
        public User ReportedUser { get; set; }
        public ReportType ReportType { get; set; }
        public PhotoBasic Photo { get; set; }
        public DateTime CreatedAt { get; set; }

    }
}

File: Pixly-FE/src/app/core/models/DTOs/Report.ts
import { PhotoBasic } from './PhotoBasic';
import { User } from './User';
import { ReportType } from './ReportType';
import { ReportStatus } from './ReportStatus';
export interface Report {
    reportedByUser: User;
    reportedUser: User;
    photo: PhotoBasic;
    reportTitle: string;
    reportMessage: string;
    createdAt: string;
    reportType: ReportType
    reportStatus: ReportStatus
}